### PR TITLE
more consistent audio instructions

### PR DESCRIPTION
**Pull request by beausmith**
While reviewing https://github.com/votingworks/bmd/pull/966 I noticed some inconsistencies with the audio instructions. This PR makes the instructions consistently state: "to do thing, use this button"

### CODE REVIEW

File: src/pages/StartPage.tsx [hunk 1]
```diff
@@ -72,7 +81,7 @@ const StartPage = () => {
               primary
               to="/contests/0"
               id="next"
-              aria-label="Select next to start voting."
+              aria-label="Press the right button to advance to the first contest."
```
Comment (eventualbuddha) [inline src/pages/StartPage.tsx]: "right button" here vs. "down arrow" and "right arrow" in `ContestPage`. Which do we prefer?

Comment (beausmith) [reply src/pages/StartPage.tsx]: Thanks. Updated to use "button" everywhere.

**Comment by beausmith** [comment]
Please always point out other inconsistencies. There's a lot of stuff to keep straight.

File: src/utils/ScreenReader.ts [hunk 2]
```diff
@@ -169,7 +169,6 @@ export class AriaScreenReader implements ScreenReader {
     }
     return description
       .replace(/ +/g, ' ')
-      .replace(/^[. ]+/g, '')
```
Comment (eventualbuddha) [inline src/utils/ScreenReader.ts]: What problem was this causing?

**Pull request merged**